app/javascript/components/upload/DifferentialExpressionStep.jsx

```diff
@@ -49,42 +49,68 @@ export function DifferentialFileUploadForm({
     <div className="row">
       <div className="col-md-12">
         <p className="form-terra">
-          Upload a file with differential expression for a particular clustering and annotation.  (Either format, note auto-detection.  Use tables themed like pre to align columns.  Move tab to top of other files.)
-          <strong>Parsed metadata and clustering files are required before uploading</strong>
+          <p>Upload differential expression (DE) files to enable users to compare genes by DE in cells grouped by type, disease, treatment, and other experimental conditions.  Use long or wide format, one file per annotation.  Comparisons can be one-vs-rest or pairwise.</p>
```

Comment:
I think it sounds a little funny to use the word "users" here since the rest of the upload wizard uses more conversational tone like "you can do this"

Rather than "...files to enable users to compare genes by DE..." maybe like "...files to enable comparing genes by DE..."